pom.xml

```diff
@@ -49,7 +49,7 @@
         <!-- keep in alphabetic order -->
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
-        <project.systemvm.template.version>4.17.2.0</project.systemvm.template.version>
+        <project.systemvm.template.version>4.18.0.0</project.systemvm.template.version>
```

Comment:
I'll need to build the systemvmtemplate and kick the smoketests/pkging. Due to missing systemvmtemplate metadata ini/checksums, build is failing.

Reply:
thanks @rohityadavcloud for the fix and building, packaging.